.github/workflows/create-release-on-tag.yml

```
@@ -22,27 +22,50 @@ jobs:
           tag_name: ${{ github.ref }}
           release_name: PXF Version ${{ github.ref }}
           body: |
-            ## 5.16.0 (11/05/2020)
+            ## 6.0.0 (03/05/2021)
 
             ### Enhancements:
 
-            - [#480](https://github.com/greenplum-db/pxf/pull/480) Enable predicate pushdown for Hive profile when accessing Parquet backed tables
-            - [#477](https://github.com/greenplum-db/pxf/pull/477) CLI: Add --skip-register flag for pxf [cluster] init
-            - [#474](https://github.com/greenplum-db/pxf/pull/474) Optimize hive metadata
-            - [#467](https://github.com/greenplum-db/pxf/pull/467) clarify the pxf.fs.basePath description
-            - [#472](https://github.com/greenplum-db/pxf/pull/472) Enable column projection for Parquet files read via Hive profile
-            - [#461](https://github.com/greenplum-db/pxf/pull/461) Increase default maximumPoolSize property in jdbc-site.xml
-            - [#469](https://github.com/greenplum-db/pxf/pull/469) Specify Hive schema column names and types in HiveAccessor when creating RecordReaders
-            - [#456](https://github.com/greenplum-db/pxf/pull/456) Add support for File Storage (Attached to every segment host)
-            - [#453](https://github.com/greenplum-db/pxf/pull/453) Remove Configuration from SessionId
-            - [#453](https://github.com/greenplum-db/pxf/pull/453) Release UGI if there is an error during the filter execution
-            - [#451](https://github.com/greenplum-db/pxf/pull/451) Support dropping columns in PXF writable external tables
-            - [#451](https://github.com/greenplum-db/pxf/pull/451) Support dropping columns in PXF readable external tables
-            - [#445](https://github.com/greenplum-db/pxf/pull/445) pxf.service.user.name is commented out by default in pxf-site.xml
+            [#404](https://github.com/greenplum-db/pxf/pull/404) Migrate PXF to Spring Boot
+            [#488](https://github.com/greenplum-db/pxf/pull/488) WASB: Add missing dependency
+            [#491](https://github.com/greenplum-db/pxf/pull/491) Remove invalid GemFireXD Profile
+            [#486](https://github.com/greenplum-db/pxf/pull/486) Serialize fragment metadata using kyro instead of json for better optimization
```

```suggestion
            [#486](https://github.com/greenplum-db/pxf/pull/486) Serialize fragment metadata using kryo instead of json for better optimization
```
this is also no longer relevant as we do not ship fragments out,  but we might start again later, so maybe it's ok to leave it